internal/idp/keycloak/authz_groups.go

```
@@ -0,0 +1,169 @@
+	for _, group := range groups {
+		if group.Path == groupPath {
+			return group.ID, nil
+		}
+	}
+
+	return "", fmt.Errorf("group not found: %s", groupPath)
+}
+
+// Helper methods for authorization permission management
+
+func (c *Client) getScopeID(ctx context.Context, clientUUID, scopeName string) (string, error) {
```

golangci-lint (2.12.2)

[error] func (*Client).getScopeID is unused

(unused)